guides/common/modules/con_configuring-freeipa-server-as-an-external-identity-provider-for-project.adoc

```diff
@@ -0,0 +1,12 @@
+[id="configuring-freeipa-server-as-an-external-identity-provider-for-project_{context}"]
```

Comment:
Similar to guides/common/modules/proc_configuring-hammer-cli-to-accept-freeipa-credentials.adoc, you can use "{FreeIPA-context}".

Reply:
Updated.